book/react-native/android/android.md

```diff
@@ -1,8 +1,11 @@
 ## React Native in Android
-As you would expect, most of the features from React Native will work out of the box for both platforms. Nevertherless,there are certaing differences between platforms which can create development friction. 
+As you would expect, most of the features from React Native will work out of the box for both platforms. Nevertherless,there are certaing differences between platforms which can create development friction.
```

Comment:
Typo in certain

Reply:
@annabauza do we know what range of react-native version we support with our current tooling ?

Reply:
Yes, we do and we can easily extend it.

Reply:
even with the breaking changes in metro-bundler / babel versions on react 55 / 56 ?